### PR TITLE
Fix the db work pool log 

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>
Resolved: https://issues.redhat.com/browse/ACM-10114

### CODE REVIEW

File: pkg/conflator/workerpool/worker_pool.go [hunk 1]
```diff
@@ -66,11 +63,14 @@ func (pool *DBWorkerPool) Start(ctx context.Context) error {
...
+	ticker := time.NewTicker(10 * time.Second)
+	defer ticker.Stop()
+
 	for i := 0; i < 6; i += 1 {
 		select {
 		case res := <-pool.workers:
 			return res, nil
-		case <-pool.ticker.C:
+		case <-ticker.C:
 			pool.log.Info("the db workers are not available, retrying", "seconds", i*10+10)
```
Comment (clyang82) [inline pkg/conflator/workerpool/worker_pool.go:74]: add log level for debug only.

Comment (yanmxa) [reply pkg/conflator/workerpool/worker_pool.go]: Done.

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**